docs/rules/no-unsafe-finally.md

```diff
@@ -0,0 +1,134 @@
+# disallow control flow statements in `finally` blocks (no-unsafe-finally)
+
+Javascript suspends the control flow statements of `try` and `catch` blocks until the execution of `finally` block finishes. So, when a control flow statement is used in a `finally` block, it overwrites the previous statements, which is considered as unexpected behavior. Examples of this situation:
+
+When `return` is used:
```

Comment:
They're not ignored. They are still evaluated. The value is just not used as the function return value.